package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@chanzuckerberg/eds",
-  "version": "12.0.2",
+  "version": "12.1.0-alpha.0",
```

Comment:
Are we pushing alpha versions?

Reply:
i pushed one for testing this on consuming projects, and will restore this to the right thing after. Also realized we need to document the step to get this to GH registry, which i will add to this PR